iocore/hostdb/Makefile.am

```diff
@@ -31,15 +31,58 @@ EXTRA_DIST = I_HostDB.h
 noinst_LIBRARIES = libinkhostdb.a
 
 libinkhostdb_a_SOURCES = \
+  I_SplitDNS.h \
+  I_SplitDNSProcessor.h \
   HostDB.cc \
   I_HostDB.h \
   I_HostDBProcessor.h \
   Inline.cc \
-  MultiCache.cc \
+  RefCountCache.cc \
   P_HostDB.h \
   P_HostDBProcessor.h \
-  P_MultiCache.h
+  P_RefCountCache.h
+
+# TODO: collapse into a single?s
+TESTS = $(check_PROGRAMS)
+check_PROGRAMS = test_RefCountCache
+
+test_RefCountCache_SOURCES = \
+  test_RefCountCache.cc
+
 
 #test_UNUSED_SOURCES = \
 #  test_I_HostDB.cc \
 #  test_P_HostDB.cc
+
+
+test_LD_FLAGS = \
```

Comment:
These are so that any other tests in this subsystem can use the same things. I do plan on adding some DNS tests down at this layer as well-- so although this is a weird lookaside for now it'll be useful in the future (not to mention that matches what the other modules do)